configs/embedded-milvus.yaml

```diff
@@ -18,7 +18,7 @@
 etcd:
   endpoints:
     - localhost:2379
-  rootPath: by-dev # The root path where data is stored in etcd
+  rootPath: /by-dev # The root path where data is stored in etcd
```

Comment:
Is there a compatibility issue? At least we need to support a fallback without /

Reply:
reverted